purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/properties/Padding.kt

```diff
@@ -26,8 +28,8 @@ internal data class Padding(
      */
     val trailing: Double,
 ) {
-    companion object {
-        val zero = Padding(0.0, 0.0, 0.0, 0.0)
-        val default = Padding(10.0, 10.0, 20.0, 20.0)
+    internal companion object {
+        internal val zero = Padding(0.0, 0.0, 0.0, 0.0)
+        internal val default = Padding(10.0, 10.0, 20.0, 20.0)
```

Comment:
do these need the internal as well? I think it will be implicit

Reply:
You used to be able to still call members of a non-`public` companion object from Java, but I think that had to do with `private` companion objects and not `internal` ones. 

However, marking the fields as `internal` is not sufficient either to hide it from Java completely, as `internal` things become public, albeit with [mangled names](https://kotlinlang.org/docs/java-to-kotlin-interop.html#visibility). (This became `getDefault$purchases_defaultsRelease()`.)

I experimented some more and went with your earlier suggestion of using `@JvmSynthetic` (https://github.com/RevenueCat/purchases-android/pull/1939/commits/94003a4528341f4615e2e8619cf267404282da50). This achieves the goal of hiding these fields from Java completely. Thanks for making me double check this!